test/unit/test_sap_cli_featuretoggle.py

```
@@ -21,7 +21,17 @@
-class TestFeatureToggle(PatcherTestCase, ConsoleOutputTestCase):
+class TestFeatureToggle(ConsoleOutputTestCase, PatcherTestCase):
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        PatcherTestCase.__init__(self)
+
+    def tearDown(self):
+        try:
+            PatcherTestCase.unpatch_all(self)
+        finally:
+           super().tearDown()
```

Flake8 (7.3.0)

[error] indentation is not a multiple of 4

(E111)